west.yml

```diff
@@ -335,8 +337,9 @@ manifest:
       revision: 150f4eb2955eaf36ac0f9519d4f4f58d5ade5740
       path: modules/lib/uoscore-uedhoc
     - name: zcbor
-      revision: dbe20afd00b3ddd6956f4b47f5df202bb49a8707
+      revision: pull/389/head
```

Comment:
I think this PR should be opened against https://github.com/zephyrproject-rtos/zcbor, unless you plan to do an upmerge of zcbor right after the referenced PR has been merged.

Reply:
Agreed. @nordicjm please open it in the Zephyr fork

Reply:
Agreed to defer it to later.